src/Contracts/TransformableData.php

```diff
@@ -20,6 +20,7 @@ public function jsonSerialize(): array;
 
     public function transform(
         bool $transformValues = true,
+        bool $mapPropertyNames = true,
```

Comment:
Can you add this behind $wrapExecutionType? That way we do not have a breaking change.

Reply:
already done